k8s/base/jicofo-deployment.yml

```diff
@@ -44,5 +44,12 @@ spec:
           readinessProbe:
             tcpSocket:
               port: 8888
+          resources:
+            limits:
+              cpu: 3300m
```

Comment:
could all these requests/limits values come from a template variable?

Reply:
Using kustomize there is no need for that. You override the part you want in your overlay. For example for this deployment you can create this is your overlay:

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: jicofo
  labels:
    app: jicofo
spec:
  template:
    spec:
      containers:
        - name: jitsi-meet
          resources:
            limits:
              cpu: 1000m
              memory: 4000Mi
            requests:
              cpu: 100m
              memory: 50Mi
```

Reply:
:+1: